test/functionalTest/cases/3658_env_vars/env_vars.test

```diff
@@ -0,0 +1,134 @@
+                               [option '-statCounters' (enable request/notification counters statistics)]          ORION_STAT_COUNTERS             FALSE /FALSE/                                        
+                               [option '-statSemWait' (enable semaphore waiting time statistics)]                  ORION_STAT_SEM_WAIT             FALSE /FALSE/                                        
+                               [option '-statTiming' (enable request-time-measuring statistics)]                   ORION_STAT_TIMING               FALSE /FALSE/                                        
+                               [option '-statNotifQueue' (enable thread pool notifications queue statistics)]      ORION_STAT_NOTIF_QUEUE          FALSE /FALSE/                                        
+                               [option '-logSummary' <log summary period in seconds (defaults to 0, meaning 'of]   ORION_LOG_SUMMARY_PERIOD        0 <= 0 /0/ <= 2678400                                
+                               [option '-relogAlarms' (log messages for existing alarms beyond the raising alar]   ORION_RELOG_ALARMS              FALSE /FALSE/                                        
+                               [option '-strictNgsiv1Ids' (additional checks for id fields in the NGSIv1 API)]     ORION_CHECK_ID_V1               FALSE /FALSE/                                        
+                               [option '-disableCustomNotifications' (disable NGSIv2 custom notifications)]        ORION_DISABLE_CUSTOM_NOTIF      FALSE /FALSE/                                        
+                               [option '-logForHumans' (human readible log to screen)]                             ORION_LOG_FOR_HUMANS            FALSE /FALSE/                                        
+                               [option '-disableMetrics' (turn off the 'metrics' feature)]                         ORION_DISABLE_METRICS           FALSE /FALSE/                                        
+                               [option '-insecureNotif' (allow HTTPS notifications to peers which certificate c]   ORION_INSECURE_NOTIF            FALSE /FALSE/                                        
+                               [option '-ngsiv1Autocast' (automatic cast for number, booleans and dates in NGSI]   ORION_NGSIV1_AUTOCAST           FALSE /FALSE/                                        
+
+
+
+02. Set the broker pidfile using env var ORION_PID_PATH=/tmp/cb.pid, start the broker in BG, make sure that /tmp/cb.pid has been created
+========================================================================================================================================
+OK - the pidfile /tmp/cb.pid has been created
+OK - the pidfile /tmp/cb.pid disappeared once the broker died
+
+
+--TEARDOWN--
```

Comment:
```suggestion
--TEARDOWN--
rm /tmp/cb3658.pid
```

To impreove cleaness in the case of fail

Reply:
Fixed in cde3f08